### PR TITLE
feat: add a setting for toggling requesting additional scopes

### PR DESCRIPTION
**Pull request by daveleek**
Adds a setting to OIDC SSO configuration that controls whether or not additional scopes are to be requested during login

Skjermbilde 2023-08-24 kl  09 00 54

### CODE REVIEW

File: frontend/src/component/admin/auth/OidcAuth/OidcAuth.tsx [hunk 1]
```diff
@@ -236,6 +241,34 @@ export const OidcAuth = () => {
                     data={data}
                     setValue={setValue}
                 />
+                <Grid container spacing={3} mb={2}>
+                    <Grid item md={5}>
+                        <strong>Enable Additional Scopes</strong>
+                        <p>
+                            If you enable additional scopes unleash will request
```
Comment (nunogois) [inline frontend/src/component/admin/auth/OidcAuth/OidcAuth.tsx:248]:
```suggestion
                            If you enable additional scopes Unleash will request
```

File: frontend/src/component/admin/auth/OidcAuth/OidcAuth.tsx [hunk 2]
```diff
@@ -236,6 +241,34 @@ export const OidcAuth = () => {
                     data={data}
                     setValue={setValue}
                 />
+                <Grid container spacing={3} mb={2}>
+                    <Grid item md={5}>
+                        <strong>Enable Additional Scopes</strong>
+                        <p>
+                            If you enable additional scopes Unleash will request
+                            the additional scopes needed to match configuration
+                            beyond openid, profile, and email. For example 'groups'.
```
Comment (gardleopard) [inline frontend/src/component/admin/auth/OidcAuth/OidcAuth.tsx:250]: I wonder if this should be add groups scope instead, because now its only groups.

Comment (daveleek) [reply frontend/src/component/admin/auth/OidcAuth/OidcAuth.tsx]: Yeah I think you're right! I've changed it to be addGroupsScope and only be for the groups scope

File: frontend/src/component/admin/auth/SsoGroupSettings.tsx [hunk 3]
```diff
@@ -76,6 +81,32 @@ export const SsoGroupSettings = ({
                     />
                 </Grid>
             </Grid>
-        </>
+            <Grid container spacing={3} mb={2}>
+                    <Grid item md={5}>
+                        <strong>Request 'groups' Scope</strong>
+                        <p>
+                            When enabled Unleash will also request the 'groups' scope as part of the login request.
```
Comment (gardleopard) [inline frontend/src/component/admin/auth/SsoGroupSettings.tsx:88]: much clearer!

**Review by gardleopard — APPROVED** [review]
Great :)

**Review by nunogois — APPROVED** [review]

**Pull request merged**